docs/release-notes/august-2024.md

```diff
@@ -0,0 +1,128 @@
+- **Semgrep Managed Scans**: Improved error messages to users when clicking **Run a new scan** from the **Projects > Details** page. Now you are better equipped to troubleshoot issues with managed scans. <!-- 16025 -->
+- Updated the Buildkite CI configuration template. <!-- 15932 -->
+- **Code search**: YAML is now validated in the search step and invalid YAML is caught when viewing results. <!-- 15886 -->
+
+### Fixed
+
+- **Jira**: Fixed a bug which prevented error messages from appearing in tooltips when Jira tickets failed to be created. Now, you can see detailed error messages letting you know what went wrong when a Jira ticket is not successfully created through Semgrep. <!-- 16259 -->
+- Fixed a regression in which clicking outside of the **Findings** page filter component did not clear all filters.
+- Various copy edits to the Dashboard (beta) page. <!-- 16176 -->
+- Fixed an issue in which untriaged findings could be marked as reopened when creating Jira tickets from the **Finding details** page. <!-- 15969 -->
+- Fixed a bug in which the **Dashboard** did not display the correct number of findings. <!-- 15935-->
+
+## 💻 Semgrep Code
+
+### Added
+
+- **Docker**: Semgrep ellipses `...` are now allowed in patterns for `HEALTHCHECK` commands.
+- **Terraform**: Added support for `.tfvars` files. <!-- SAF-1481 -->
+
+### Changed
+
+- Semgrep CLI's `--debug` flag no longer generates profiling information, including time and scan performance measurements. To obtain this information, use `--time`.
+
+### Fixed
+
+- Fixed an error with Julia list comprehensions. For example, the pattern `[$A for $B in $C]` matches `[x for y in z]` and result in three bindings `[$A/x,$B/y,$C/z]` instead of one `[$A/x]`.
+- Fixed an issue resulting in deadlock when a scan has interfile analysis and tracing enabled and the number of subprocesses is greater than 1 (`j < 1`). <!-- SAF-1157 -->
+- Fixed an issue where the number of files reported as scanned by Semgrep CLI was inflated due double-counting of generic and regex modes. <!-- SAF-507 -->
+- `--debug` now generates fewer log entries. Additionally, when the number of ignored files, rules, or other entities is too large, Semgrep indicates this in the logs with `<SKIPPED DATA>` to keep the output minimal.
+
+## ⛓️ Semgrep Supply Chain
+
+### Added
+
+- You can now filter and view EPSS scores for your Supply Chain findings.
+
+### Changed
+
+- The link to the Supply Chain findings page in Semgrep AppSec Platform filters to the specific repository and `ref` on which the findings are detected.
+
+### Fixed
+
+- Fixed an issue where Supply Chain's Findings Detail pages weren't showing detailed error information.
+
+## 🤖 Semgrep Assistant
+
+### Added
+
+- Assistant Memories is now in **public beta**. Assistant Memories allows you to tailor Assistant's remediation guidance to your organization's standards and defaults on a per-project, per-rule basis.
```

Comment:
```suggestion
- Assistant Memories is now in **public beta**. It allows you to tailor Assistant's remediation guidance to your organization's standards and defaults on a per-project, per-rule basis.
```
Feel free to ignore. The use of common nouns as proper nouns... it can't be helped (Teams, Managed Scans, etc) but maybe we can use a pronoun (not too hot on this idea either).

Reply:
What about side-stepping this using "this feature?"

Reply:
Yes that sounds great!